ui/src/picker.test.tsx

```diff
@@ -41,6 +41,25 @@ describe('Picker.tsx', () => {
     expect(queryByTestId(name)).toBeInTheDocument()
   })
 
+  it('Does not render data-test attr when max choices reached', () => {
```

Comment:
Not sure I understand these tests. We do want to render `data-test` every time a picker is rendered, irrespective of `max_choices`. `data-test_remove` should be rendered every time a value is selected (has a cancel button).

Reply:
You're right, `max_choices` is irrelevant here since the request is about introducing a `data-test` attribute on the picker's remove button. I have updated the test to remove references to `max_choices`.